wallets/config.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
what this line is for?

Reply:
I have no idea. It's the same in `services/chain/config.yml`